README.md

```diff
@@ -10,11 +10,16 @@ Go clients for talking to a [kubernetes](http://kubernetes.io/) cluster.
 * The `transport` package is used to set up auth and start a connection.
 * The `tools/cache` package is useful for writing controllers.
 
-### Releases
+### Versions
```

Comment:
These [lines](https://github.com/niemeyer/gopkg/blob/master/main.go#L228-L239) redirect the clients to checkout a specific version. As lavalamp said, we can fork our gopkg.in.

([Doc](https://git-scm.com/book/en/v2/Git-Internals-Transfer-Protocols) on git protocol)